hindi/art-of-living/who_developed_hindi.md

```diff
@@ -0,0 +1 @@
+Hindi (Hindi: हिन्दी hindī), sometimes spelled हिंदी, also called Modern Standard Hindi (Hindi: मानक हिन्दी mānak hindī), is a standardised and Sanskritised register of the Hindustani language. Hindi is one of the many official [[official languages |language]] of the Union of India.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.